backend/app/views/spree/admin/shared/_tabs.html.erb

```diff
@@ -3,7 +3,9 @@
 <% end %>
 
 <% if can? :admin, Spree::Product %>
-  <%= tab *Spree::BackendConfiguration::PRODUCT_TABS, icon: 'th-large' %>
+  <%= tab *Spree::BackendConfiguration::PRODUCT_TABS, icon: 'th-large' do %>
+    <%- render partial: 'spree/admin/shared/product_sub_menu' %>
```

Comment:
Since there's now some confusion about what a "sidebar" is (left side of the page vs right side of the main content area), I'm thinking of calling all of the new stuff "admin navigation". But yes, grouped in a folder would be nice.